test/sequence/smart/token_refresh_sequence_test.rb

```diff
@@ -0,0 +1,213 @@
+# frozen_string_literal: true
+
+require_relative '../../test_helper'
+
+describe Inferno::Sequence::TokenRefreshSequence do
```

Comment:
Compare to this more integration-style test which requires that the entire sequence be run for each test and depends on inspecting a `TestResult` object to check the outcome: https://github.com/onc-healthit/inferno/blob/master/test/sequence/token_refresh_sequence_test.rb

Reply:
This looks good. Possible to DRY out this repeated block:

`before do
  @test = SEQUENCE[SOME_TEST_ID]
  @sequence = SEQUENCE.new(@instance, @client)
end`